base/setup/reactos/reactos.h

```diff
@@ -108,6 +115,15 @@ typedef struct _NT_WIN32_PATH_MAPPING_LIST
 } NT_WIN32_PATH_MAPPING_LIST, *PNT_WIN32_PATH_MAPPING_LIST;
 
 
+#if 0
+typedef struct _KBLAYOUT
+{
+    TCHAR LayoutId[9];
+    TCHAR LayoutName[128];
+    TCHAR DllName[128];
+} KBLAYOUT, *PKBLAYOUT;
+#endif
+
```

Comment:
you moved that down for now obvious reason and therefore make the PR larger than necessary. I do recommend to move it up again.

Reply:
The reason is obvious to me, though....